geo/map_processing/map_processor.py

```diff
@@ -21,14 +21,16 @@
 
 FLAGS = flags.FLAGS
 flags.DEFINE_string("place", None, "map area - Manhattan or Pittsburgh.")
```

Comment:
Let's use an enum here with options 'manhattan' and 'pittsburgh'-- see DEFINE_enum: https://abseil.io/docs/python/guides/flags

Reply:
done